### ISSUE TITLE
Bad time displaed after update to 4.0

### ISSUE THREAD

**Issue by bgga** [Blackymas/NSPanel_HA_Blueprint#1092]
Hi.
After updating nspanel to 4.0 the time is displayed wrog. 
My timezone is Europe/Warsaw and the time on nspanel is -2h than my time zone.
Before update was good.
HA ver: 2023.9.2
ESP Home ver: 2023.8.3

Thanks

**Comment by edwardtfn** [comment]
We moved the time engine from Blueprint to ESPHome, so time keeps updated even when network is out.
ESPHome should get the timezone from your server where you compiled the code, however, if this is not working properly, you can Enforce the timezone by adding this to your ESPHome settings (at the end of the file is OK):

```yaml 
time:
  - id:!extend time_provider
    timezone: "America/Cancun"
```

**Comment by edwardtfn** [comment]
@bgga, have that suggestion solved the issue you reported?

**Comment by cis202** [comment]
Adding the above suggested to my docker-compose.yml file, section esphome, did NOT solve the issue.
Since upgrading to v4.1.4, time shown on the nspanels is UTC instead of Europe/Brussels.
In ESPHome container, running date gives correct time, but cat /etc/timezone shows 'Etc/UTC'.
Also tried adding 
- /etc/timezone:/etc/timezone:ro
- TZ="Europe/Brussels"
all on top of /etc/localtime:/etc/localtime:ro, as it was initially.
There are a lot of issues around the net about esphome and time, so I don't know if moving the engine was a good idea.

**Comment by edwardtfn** [comment]
You should have added that to your panel's yaml.

**Comment by edwardtfn** [comment]
Here you will find instructions for customizations: https://github.com/Blackymas/NSPanel_HA_Blueprint/wiki/(EN)-Customization#instructions

**Comment by fdemmer** [comment]
The suggested timezone override fixed the displayed time for me with v4.1.4 :pray:

**Comment by cis202** [comment]
Adding this to the yaml file of the device did the trick. Thanks a lot.
Sidenote: needed to add an extra space between id: and !extend though.
